SASS/1-blocks/_nav.scss

```diff
@@ -0,0 +1,52 @@
+header{
+     background:white;
+     text-align:center;
+     position: absolute;
+     z-index: 999;
+    width:100%;
+}
+nav{
+    background: white;
+     width:100%;
+    position: absolute;
+    text-align: left;
+    top: 100%;
+    left: 0;
+     display: none;
+    // transform: scale(1,0);
+    // transform-origin: top;
+    // transition: transform 400ms ease-in-out;
+    ul{
+        margin: 0;
+        padding: 0;
+    }
+    li{
+        margin-bottom: 1rem;
+        margin-left: 1rem;
+        list-style: none;
+    }
+    a{
+        color:$lightColor;
+        text-decoration: none;
+        &:hover{
+            color:rgb(143, 6, 6);
+        }
+    }
+}
+.nav-toggle{
+     display: none;
+    &:checked ~nav{
+         display: block;
+// transform: scale(1,1);
+// text-transform: uppercase;
+// font-weight: bolder;
```

Comment:
`REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.